Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -58,14 +58,20 @@ extension Endpoint {
             body: body
         )
     }
-
-    static func muteChannel(cid: ChannelId, mute: Bool) -> Endpoint<EmptyResponse> {
-        .init(
+    
+    static func muteChannel(cid: ChannelId, mute: Bool, expiration: Int? = nil) -> Endpoint<EmptyResponse> {
+        var body: [String: AnyEncodable] = ["channel_cid": AnyEncodable(cid)]
```

Comment:
ChannelId is already Codable. i don't think there is a need for the wrap to AnyEncodable.

Reply:
Hmm, you sure? 🤔
<img width="1316" alt="Screenshot 2024-03-14 at 11 23 13 AM" src="https://github.com/GetStream/stream-chat-swift/assets/20794991/6636576f-d778-408a-adc2-662a426bbb6c">

Reply:
Yeah this is because the expiration and the channelId have different types so the dictionary doesn't know how to setup. You could use `any Encodable` potentially (which is Swift's way to solve the issue that our AnyEncodable tries to fix) but that may require further changes. Sorry for confusing you here. You can ignore me.